packages/interactive_media_ads/lib/src/platform_interface/ui_element.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Describes an element of the ad UI, to be requested or rendered by the SDK.
+enum UIElement {
```

Comment:
This seems like a very generic name that could end up require people to use prefixing; what about `AdUIElement`?